README.md

```diff
@@ -355,15 +355,15 @@ FLAGS
                                <options: goerli|mumbai|fuji>
   --tokenId=<value>            [default: 0] The token id to mint. By default the token id is 0, which mints the next
                                available token id
-  --tokenUri=<value>           The uri of the token, minus the prepend (ie "ipfs://")
-  --tokenUriType=<option>      The token URI type
+  --uri=<value>           The uri of the token, minus the prepend (ie "ipfs://")
+  --uriType=<option>      The token URI type
                                <options: ipfs|https|arweave>
 
 DESCRIPTION
   Mint a Holographable NFT.
 
 EXAMPLES
-  $ holograph create:nft --network="goerli" --collectionAddress="0xf90c33d5ef88a9d84d4d61f62c913ba192091fe7" --tokenId="0" --tokenUriType="ipfs" --tokenUri="QmfQhPGMAbHL31qcqAEYpSP5gXwXWQa3HZjkNVzZ2mRsRs/metadata.json" --env mainnet|testnet|develop|experimental
+  $ holograph create:nft --network="goerli" --collectionAddress="0xf90c33d5ef88a9d84d4d61f62c913ba192091fe7" --tokenId="0" --uriType="ipfs" --uri="QmfQhPGMAbHL31qcqAEYpSP5gXwXWQa3HZjkNVzZ2mRsRs/metadata.json" --env mainnet|testnet|develop|experimental
```

Comment:
@alexanderattar so why did you change the name? I think we can change it to be clearer. I just want to make sure people don't confuse the image CID and the metadata CID. 

Also if we make this change, we have to re-generate the readme file.

Reply:
I updated the CLI code to reflect these changes so I believe if we regenerate the readme it should stay the same, but we can try that out

Reply:
I changed the name to acknowledge that this flag can be used to send and validate not just tokenUris but other uses such as the drop metadata uri. We can duplicate the logic to keep things more specific if that's preferred. lmk what you think